src/test/java/com/ZEROBANK/pages/BasePage.java

```diff
@@ -26,6 +27,15 @@
     public WebElement onlineStatementsTab;
 
 
+    public void navigateToTab(String tabName){
+     Driver.get().findElement(By.xpath("//a[text()='"+tabName+"+']")).click();
+    }
+
+    public void navigateToAccountType(String accountType){
+        Driver.get().findElement(By.xpath("//a[text()='"+accountType+"+']")).click();
```

Comment:
Hay, this method has a similar one in LoginPAge. Instead of creating a new one, just call that method.m